tools/schemas/app-config.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "SGX-LKL app config schema",
+    "required": [
+        "run",
+        "args",
+        "disk_config"
+    ],
+    "additionalProperties": false,
+    "properties": {
+        "run": {
+            "type": "string",
+            "description": "Absolute path to the application",
+            "pattern": "^/.+$"
+        },
+        "args": {
+            "type": "array",
+            "description": "Application arguments",
+            "default": [],
+            "items": {
+                "type": "string"
+            }
+        },
+        "cwd": {
+            "type": "string",
+            "description": "Absolute path to the working directory",
```

Comment:
What's the default of cwd if not specified?

Reply:
That would be `/`, I'll add it.